forte/data/batchers.py

```diff
@@ -84,32 +84,34 @@ def flush(self) -> Iterator[Dict]:
 
         """
         if self.current_batch:
-            yield self.current_batch
+            yield self.data_pack_pool, None, self.current_batch
```

Comment:
I guess the return value type should change? Will be helpful to add what are these 3 in the `Returns:` doc too.

Reply:
Also good to mention why it is always `None` (i.e. ignored here).

Reply:
Got it.